ios/AllAboutOlaf/AppDelegate.m

```diff
@@ -14,6 +14,11 @@
 #import <React/RCTBridge.h>
 #import <React/RCTBundleURLProvider.h>
 #import <React/RCTRootView.h>
+#if __has_include(<React/RNSentry.h>)
+#import <React/RNSentry.h> // This is used for versions of react >= 0.40
+#else
+#import "RNSentry.h" // This is used for versions of react < 0.40
+#endif
```

Comment:
Remove the else branch

Reply:
And by proxy, leave only the content of the if - there's no need for the conditional here at all, since we're running 0.59

Reply:
Okay, yeah.